### PR TITLE
updates to dir2cast.

### PR DESCRIPTION
**Pull request by jeffbearer**
Here is the pull request you asked for.  if you need me to do / change anything let me know.

### CODE REVIEW

**Comment by marctytus** [comment]
Hi @ben-xo - just wondering why you didn't merge this pull request.

File: dir2cast.php [hunk 1]
```diff
@@ -945,6 +991,11 @@ public function init()
 	 */
 	protected function acquireLock()
 	{
+		if (!file_exists($this->temp_dir))
+		{
+			mkdir($this->temp_dir, 0755, true);
```
Comment (ben-xo) [inline dir2cast.php]: This should probably throw a new Exception if it fails.

**Comment by ben-xo** [comment]
Hi @marctytus - no, I just completely forgot about it! I will merge. The changes look great.

**Comment by jeffbearer** [comment]
👍  Thanks.

**Comment by marctytus** [comment]
Woo! 👍

**Pull request merged**